src/_sass/color_schemes/centrapay-theme.scss

```diff
@@ -192,25 +187,10 @@ a:not([class]) {
     $chocolate-600 0%,
     $chocolate-600 100%
   ) !important;
-
   &:hover {
     background-image: linear-gradient(
       $neutral-900 0%,
       $neutral-900 100%
     ) !important;
   }
 }
-
-// Other things...
-
-
-// $table-background-color: $white !default;
-// $code-background-color: $grey-lt-000 !default;
-// $feedback-color: darken($sidebar-color, 3%) !default;
-// $body-text-color: $grey-dk-100 !default;
-// $body-heading-color: $grey-dk-300 !default;
-// $search-result-preview-color: $grey-dk-000 !default;
-// $nav-child-link-color: $grey-dk-100 !default;
-// $link-color: $purple-000 !default;
-// $btn-primary-color: $purple-100 !default;
-// $base-button-color: #f7f7f7 !default;
```

Comment:
If it's dead, let it die. You can look these up when you need them. They're not useful here.